.github/linters/ct.yaml

```diff
@@ -0,0 +1,10 @@
+# See https://github.com/helm/chart-testing#configuration
+remote: origin
+target-branch: integration
+helm-extra-args: "--timeout 600s"
+chart-dirs:
+  - k8_cortx_cloud/charts
+excluded-charts:
+chart-repos:
+validate-chart-schema: false
+validate-maintainers: false
```

Comment:
Do we have an email address to set for maintainers? I think this is for Chart.yaml `maintainers`.

Reply:
I think that would be a Bent / community discussion point. @walterlopatka is that something you want to run down from a logistics point of view? We can either do that individually for everyone who is working on the chart or we can have a single roll-up maintainer group that is the squad, etc?

For reference: https://helm.sh/docs/topics/charts/#the-chartyaml-file

Reply:
I asked Bent and recommended using cortx-questions@seagate.com.  (He said he would add me to that list.)